lib/salus/scanners/osv/gradle_osv.rb

```diff
@@ -12,7 +12,7 @@ class SemVersion < Gem::Version; end
         "/Maven/all.zip".freeze
 
     def should_run?
-      @repository.build_gradle_present?
+      @repository.build_gradle_present? && File.exist?('/home/repo/gradlew')
```

Comment:
I think `File.exist?('/home/repo/gradlew')` should be moved to the `run` method.
What do you think, @maitrayshah ?

Reply:
Also, Please replace all occurrences`/home/repo` with `@repository.path_to_repo`.
If I'm not mistaken, you can replace `/home/repo` with something else with docker command line.

Reply:
Also, Please replace all occurrences of`/home/repo` with `@repository.path_to_repo`.
If I'm not mistaken, you can replace `/home/repo` with something else with docker command line.